templates/insights/blocks/promoted_list_block.html

```diff
@@ -4,14 +4,6 @@
 
 <div class="related-resources" data-container-name="related-resources">
     <div class="container">
-        {% if value.category.icon %}
-        {% static value.category.icon_static_path as bg_image %}
-        <div class="promoted-item__icon" style="background-image: url('{{ bg_image }}');"></div>
-        {% endif %}
-        <p class="related-resources__icon-label" role="presentation" aria-hidden="true">
-            {{ value.category.name }}
-        </p>
-
```

Comment:
Is this branch up-to-date with the latest `develop`? I swear I already removed this :S

Reply:
I rebased earlier as I was missing a few commits but they were for some work Ash merged yesterday. So it should have all your latest changes :S